apps/meteor/server/features/EmailInbox/EmailInbox_Outgoing.ts

```diff
@@ -36,8 +37,8 @@ const sendErrorReplyMessage = (error: string, options: any): void => {
 	sendMessage(user, message, { _id: options.rid });
 };
 
-function sendEmail(inbox: Inbox, mail: Mail.Options, options?: any): void {
-	inbox.smtp
+function sendEmail(inbox: Inbox, mail: Mail.Options, options?: any): Promise<any> {
```

Comment:
> `any`

![image](https://user-images.githubusercontent.com/2263066/185689329-2ba6ce43-7023-40a9-b371-a227b8f45c16.png)

Reply:
i was just passing along the type from the lib, but yeah, it's better as {messageId: string}